### PR TITLE
Don't allow PHP 7 to fail

### PR DESCRIPTION
**Pull request by GrahamCampbell**
PHP 7.0 is RC now. It's pretty much fixed. At this point, I'd say we really want to know if things start failing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,9 @@ php:
     - hhvm
 
 matrix:
-    fast_finish: true
```
Comment (gharlan) [inline .travis.yml]: why?

Comment (GrahamCampbell) [inline .travis.yml]: fast finish is pointless if you don't allow anything to fail - the only thing it does is mark the build as passed if everything but the build that's allowed to fail is done

Comment (gharlan) [inline .travis.yml]: No.

http://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing

> Now, a build will finish **as soon as a job has failed**, or when the only jobs left allow failures.

Comment (GrahamCampbell) [inline .travis.yml]: Oh, right.

**Comment by gharlan** [comment]
:+1:

**Comment by keradus** [comment]
Our PHP CS Fixer may be run under PHP7 and it's work really great, but it can't fix PHP7 code yet.
So one may use tool under PHP7 to fix code without PHP7 features, which is sad and I'm not sure do we really want to say "we support PHP7" before we really support PHP7 syntax...

**Comment by GrahamCampbell** [comment]
I don't really agree here. What we're saying is "our codebase runs on php 7 and can fix php 5 code just fine as our tests prove".

**Comment by keradus** [comment]
Fine for me ;)

**Comment by keradus** [comment]
Thank you @GrahamCampbell.

**Pull request closed without merging**